plugins/package-managers/node/src/test/kotlin/utils/NpmSupportTest.kt

```diff
@@ -192,17 +193,16 @@ class NpmSupportTest : WordSpec({
 
     "parseNpmVcsInfo()" should {
         "get VCS information from an object node" {
-            @Suppress("Wrapping")
-            val node = ObjectMapper().run {
-                createObjectNode().apply {
-                    replace("gitHead", TextNode("bar"))
-                    replace("repository", createObjectNode().apply {
-                        replace("type", TextNode("Git"))
-                        replace("url", TextNode("https://example.com/"))
-                        replace("directory", TextNode("foo"))
-                    })
+            val node = """
```

Comment:
It'd be some work, but I guess ideally this should be done for all tests in this file now, or? That would nice decouple the test from Jackson as well.

Reply:
I would do a follow up PR for this

Reply:
here you go: https://github.com/oss-review-toolkit/ort/pull/8518